src/SimpleThings/EntityAudit/RevisionCriteria.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace SimpleThings\EntityAudit;
+
+class RevisionCriteria
+{
+    protected $dateFrom;
+    protected $dateTo;
+    protected $username;
```

Comment:
This shouldn't work right now, those properties need to be public in order to access them as above `$criteria->dateFrom`